### PR TITLE
 feat(ci): Run Sentry's Symbolicator integration tests

### PR DESCRIPTION
**Pull request by armenzg**
When we make changes to Symbolicator, we should test against the latest Sentry. Up until now, we would have to merge the change to Symbolicator's `master` branch and then open a PR on Sentry to see if anything would break.

This change ensures that every PR and `master` change is tested against the latest Sentry.

#skip-changelog

### CODE REVIEW

**Comment by armenzg** [comment]
I believe this is working:
https://github.com/getsentry/symbolicator/runs/3116790297

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -115,6 +115,69 @@ jobs:
...
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Cache rust cargo artifacts
+        uses: swatinem/rust-cache@v1
+
+      - name: Build Relay
```
Comment (billyvg) [inline .github/workflows/ci.yml:136]:
```suggestion
      - name: Build Symbolicator
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -115,6 +115,69 @@ jobs:
...
+        run: cargo build --locked
+
+      - name: Run Relay
```
Comment (billyvg) [inline .github/workflows/ci.yml:139]:
```suggestion
      - name: Run Symbolicator
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -115,6 +115,69 @@ jobs:
...
+          path: ${{ steps.setup.outputs.pip-cache-dir }}
+          key: sentry-deps-${{ hashFiles('sentry/requirements**.txt') }}
+          restore-keys: sentry-deps-
+
+      - name: Setup Sentry
+        uses: ./.github/actions/setup-sentry
+        with:
+          workdir: sentry
+          snuba: true
+          kafka: true
```
Comment (billyvg) [inline .github/workflows/ci.yml:174]: Does symbolicator need these?

Comment (armenzg) [reply .github/workflows/ci.yml]: Let us try. I grabbed it from Sentry's workflow.

Comment (flub) [reply .github/workflows/ci.yml]: symbolicator itself does not use kafka at all.  it only receives http requests from the `symbolicate_event` celery task.

Comment (armenzg) [reply .github/workflows/ci.yml]: It definitely did not like the change and run for 6 hours:
image

Comment (armenzg) [reply .github/workflows/ci.yml]: We need both services:
image

**Review by billyvg — APPROVED** [review]
LGTM, let's get a final approval stamp from a symbolicator owner though

**Review by flub — APPROVED** [review]

**Comment by flub** [comment]
Looks good, thanks for doing this work for us!

**Pull request merged**